polaris-react/src/components/ContentBlock/ContentBlock.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import {classNames} from '../../utilities/css';
+
+import styles from './ContentBlock.scss';
+
+type Width = 'medium' | 'large';
```

Comment:
```suggestion
type Width = 'md' | 'lg';
```

To align with our naming conventions.